common/djangoapps/enrollment/views.py

```diff
@@ -634,6 +635,8 @@ def post(self, request):
                 }
             )
         except CourseEnrollmentExistsError as error:
+            log.debug('An enrollment already exists for user [%s] in course run [%s].',
+                      username, course_id)
             return Response(data=error.enrollment)
         except CourseEnrollmentError:
             return Response(
```

Comment:
I agree. I think having the stack trace here would make sense, since we don't necessarily know what failed in this case.